src/groups/general_linear.jl

```diff
@@ -246,6 +246,27 @@ project(::GeneralLinear, p, X) = X
 project!(::GeneralLinear, q, p) = copyto!(q, p)
 project!(::GeneralLinear, Y, p, X) = copyto!(Y, X)
 
+
+@doc raw"""
+    Random.rand(G::GeneralLinear; kwargs...0)
```

Comment:
```suggestion
    Random.rand(G::GeneralLinear; vector_at=nothing, kwargs...)
```

This way the default is mentioned (and I removed a typo)